com.visualstudio.code.json

```diff
@@ -0,0 +1,139 @@
+                "sha256": "51817e0530961975d9513b773960b4edd275f7d5c72293d5a151ed4f42aeb16a"
+            }
+        ]
+    },
+    {
+        "name": "git",
+        "make-args": ["NO_PERL=1"],
+        "make-install-args": ["NO_PERL=1"],
+        "cleanup": ["/share/git-gui"],        
+        "sources": [
+            {
+                "type": "archive",
+                "url": "https://www.kernel.org/pub/software/scm/git/git-2.15.0.tar.xz",
+                "sha256": "107116489f10b758b51af1c5dbdb9a274917b0fb67dc8eaefcdabc7bc3eb3e6a"
+            }
+        ]
+    },
+    {
+        "name": "nodejs",
+        "cleanup": [
+            "/include", 
+            "/share",
+            "/app/lib/node_modules/npm/changelogs", 
+            "/app/lib/node_modules/npm/doc",
+            "/app/lib/node_modules/npm/html", 
+            "/app/lib/node_modules/npm/man",
+            "/app/lib/node_modules/npm/scripts"
+        ],                
+        "sources": [
+            {
+                "type": "archive",
+                "url": "https://nodejs.org/dist/v7.9.0/node-v7.9.0.tar.xz",
+                "sha256": "a569764b884929f31a0772600a5cf36048ad42aa8c54ba4cabea6c067ef96a29"
+            }
+        ]
+    },
+    {
+        "name": "vscode",
+        "buildsystem": "simple",
+        "build-commands": [
+            "install apply_extra /app/bin",
+            "install code.sh /app/bin/code",
+            "cp /usr/bin/ar /app/bin",
+            "install -m644 npmrc /app/etc"            
+        ],
+        "cleanup": [
+            "/share/lintian",
+            "/share/pixmaps"
+        ],
+        "sources": [
+            {
+                "type": "script",
+                "dest-filename": "apply_extra",
+                "commands": [
+                    "ar x code.deb",
+                    "rm -f code.deb",                    
+                    "tar -xf data.tar.xz",
+                    "rm -f control.tar.xz data.tar.xz debian-binary",                    
+                    "mv usr/* .",
+                    "rmdir usr",
+                    "install -Dm644  share/pixmaps/code.png export/share/icons/hicolor/512x512/apps/com.visualstudio.code.png",
+                    "install -Dm644  share/applications/code.desktop export/share/applications/com.visualstudio.code.desktop",                
+                    "desktop-file-edit --set-key=Exec --set-value='code %F' export/share/applications/com.visualstudio.code.desktop",
+                    "desktop-file-edit --set-key=Icon --set-value='com.visualstudio.code' export/share/applications/com.visualstudio.code.desktop",                    
+                    "mkdir -p export/share/appdata",
+                    "sed -e s/code.desktop/com.visualstudio.code.desktop/ -e 's/Multiple, see /LicenseRef-proprietary=/' share/appdata/code.appdata.xml > export/share/appdata/com.visualstudio.code.appdata.xml"        
```

Comment:
> 's/Multiple, see /LicenseRef-proprietary=/'

`s/.../.../g` since it has multiple matches

Reply:
thanks!